src/types.cpp

```diff
@@ -23,12 +23,18 @@ std::ostream &operator<<(std::ostream &os, const SizedType &type)
   }
   if (type.is_pointer)
     os << "*";
+  if (type.type == Type::integer && (type.size != 8 || type.is_signed)) {
```

Comment:
I think we can use a consistent formatting for all ints, no need to have an exception for signed 64-bit ints.

```suggestion
  if (type.type == Type::integer) {
```